app/templates/components/forms/user-profile-form.hbs

```diff
@@ -52,10 +52,24 @@
       </div>
       <div class="field">
         <label class="details">{{t 'Phone'}}</label>
-        <Input
-          @type="text"
-          @name="phone"
-          @value={{this.user.contact}} />
+        <div class="ui left action input">
+          <UiDropdown @class='ui selection dropdown'>
+            <i class="dropdown icon"></i>
+            <div class="default text">{{t 'Country Code'}}</div>
+            <div class="menu">
+              {{#each countries as |country|}}
+                <div class="item">
+                  <i class="{{lowercase country.code}} flag"></i>
+                  {{country.phone}}
```

Comment:
Is the country code getting retained in DB with the user phone number? We would like to store the country code as well.
/cc @iamareebjamal